AngularJS/scss/dashboard.scss

```diff
@@ -0,0 +1,93 @@
+@mixin book-text($color, $fontSize) {
+  color: $color;
+  font-size: $fontSize;
+  line-height: $line-height-base;
+}
+
+%robotoRegular {
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 400;
+}
+
+%robotoItalic {
+  @extend %robotoRegular;
+  font-style: italic;
+}
+
+%robotoBold {
+  @extend %robotoRegular;
+  font-weight: 700;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  margin: 20px auto;
+  max-width: 700px;
+}
+
+.book-search {
+  align-items: center;
+  display: flex;
+  margin: 20px 0;
+}
+
+.search-input {
+  background-color: $wild-sand ;
+  @extend %robotoItalic;
+  height: 40px;
+  margin: 10px 0 10px 15px;
+  padding: 10px;
+  width: 240px;
+}
+
+.search-img {
+  background: $atlantis url('assets/search.svg') no-repeat center ;
+  height: 40px;
+  width: 40px;
+}
+
+.custom-select {
+  background-image: linear-gradient(45deg, transparent 50%, $dusty-gray 50%),
+  linear-gradient(135deg, $dusty-gray 50%, transparent 50%),
+  linear-gradient(to right, $wild-sand, $wild-sand);
+  background-position: calc(100% - 20px) calc(1em + 2px),
+  calc(100% - 15px) calc(1em + 2px),
+  100% 0;
+  background-repeat: no-repeat;
+  background-size: 5px 5px,
+  5px 5px,
+  2.5em 2.5em;
+  @extend .search-input;
+  @include book-text($dusty-gray, $tiny-font-size);
+}
+
+.book-list {
+  display: grid;
+  grid-gap: 50px;
+  grid-template-columns: repeat(4,150px);
+  justify-content: center;
+  margin: 20px auto;
+}
+
+.book {
+  padding: 10px;
+}
+
+.book-cover {
+  background: $alto url('assets/default_book.svg') no-repeat center ;
+  height: $book-height;
+  width: $book-width;
+}
+
+.book-title {
+  @extend %robotoBold;
+  font-family: Roboto-bold;
```

Comment:
If this extends the `robotoBold`, I think the font-family should be in that template

Reply:
Of course 
![image](https://user-images.githubusercontent.com/11933480/62958174-8bec2d80-bdcc-11e9-8b12-d3fc1b9f9f80.png)